README.md

```diff
@@ -83,4 +155,113 @@ command {
+```
+These patch files will be used to create the proper versioned WDL file at runtime (This will take priority over the automatic WDL version conversion).
+
+## Running Performance Tests
+The default runner `run.py` only runs tests and outputs them onto the commandline.
+To store the runtime of tests (and eventually graph them), the script `run_performance.py` should be used:
+```commandline
+python run_performance.py --runners toil-wdl-runner,miniwdl --output performance_output.csv --progress
+```
+The output will always be in CSV format. `--runners` can specify runners to run with in a comma separated list, and `--all-runners` is a shortcut to specify all of them.
+
+### Performance Testing Options
+[All options](#options) from the normal `run.py` script apply too. For example, if `--id stdout` is provided, then only the `stdout` test will be measured.
+
+The performance test specific options are:
+```commandline
+Arguments for running WDL performance tests:
+  --output OUTPUT, -o OUTPUT
+                        Specify the output CSV file.
+  --all-runners, -a     Specify whether to run with all runners. This will override the --runners argument.
+  --runners RUNNERS     Specify multiple runners in a comma separated list.
+```
+
+## Graphing Performance Tests
+The script `create_graph.py` can be used to graph the output of the performance tests:
+```commandline
+python create_graph.py -f performance_output.csv
+```
+If no input CSV file is given, then the program will run the entire performance test suite first.
+### Graph Options
+Similar to the performance testing script, [all options](#options) from the base runner are carried through. However, there are graphing specific options:
+```commandline
+Arguments for graphing runtimes of WDL tests:
+  --from-file FILE, -f FILE
+                        Specify a csv file to read from.
+  --display-num DISPLAY_NUM, -d DISPLAY_NUM
+                        Specify the number of tests to display per graph.
+  --display-all, -a     Display all tests on a single graph. Overrides --display-num.
+  --ignore-runner IGNORE_RUNNER
+                        Specify a runner(s) to ignore in the graph output.
+  --precision PRECISION
+                        Specify the precision when outputting float values. Ex: Default=0 will result in 1 for float value 1.4...
+  --no-labels           Specify to not display extra labels on the graph.
+  --graph-type GRAPH_TYPE
+  --conformance-file [CONFORMANCE_FILE]
+                        Specify the conformance file to read from. This will specify whether to grab/graph tests by conformance file or by CSV file test IDs only. Specifying this will make the graph accept -n, -t, -id and
+                        other related arguments.
+```
+By default, separate graphs will be created for every 30 tests.
+
+This can be controlled with `--display-num`. This overrides the number of tests to display per graph, and `--display-all` forces all tests in a single graph.
```

Comment:
```suggestion
This can be controlled with `--display-num`, which overrides the number of tests to display per graph. `--display-all` forces all tests in a single graph.
```